### ISSUE TITLE
CSS fix for Footer Nav style

### ISSUE THREAD

**Issue by JoshuaCharlesByrd** [SSYC-WebTeam/SSYCmaster#12]
I have registered the footer nav but the style is wonky for the subpage links below the "Membership" and "Boating" menus. I have tried to de bug but can't get it right now. CSS needs to be fixed.

**Comment by maureencrist** [comment]
Thanks Josh. Will work on this on Tuesday.

**Comment by JoshuaCharlesByrd** [comment]
Cool,

Great! I will be working later too but you can now sync your code and view
what we have on the staging server @ 54.148.24.114.

Remember to have your staging server setup with pages meaning you must have
your pages setup through the dashboard then you will be able to sync code
and the registered main and footer menu will appear in your localhost
dashboard. Then set menu up in your dashboard.

If you have any trouble let me know!

**Comment by maureencrist** [comment]
I registerd all three menus (main-menu, toggle-menu, footer-menu) in my functions.php file and sync'd them. I also added the correct code to header.php and footer.php to call the correct menus and uploaded them. I added pages in my local dashboard and linked them to the menus. Everything looked good and then my http://localhost:8080 stopped showing the site to me when I clicked "visit site" at the top of the dashboard. Can't figure out how to get that working again to see what the changes look like and ensure they are all working.

Also uploaded images to the media library, so the burgee flag should be working now too. Again, can't check because my http://localhost:8080 is giving me a server error.
